### DISCUSSION TITLE
Best way to add separators to the menu

### DISCUSSION THREAD

**Discussion by murrant** [mkdocs/mkdocs#3643] [Q&A]
I tried updating my theme and adding `- separator:` to my mkdocs.yml config file, but I get the following.

```
Line in 'page' config is of type <type 'dict'>, list or string expected for sub pages. Config: {u'separator': None}
```

What is the best way to specify a separator in the config file so I can process it in my theme?

**Comment by waylan** [comment]
We currently do not support "separators". You can nest your menu items to group them, but each item in the `pages` config setting must either be an actual page, or a group which contains nested pages (the config validation enforces this, which is why you're getting the error). If you're building your own templates, perhaps you could use the existing nesting feature to simulate separators (just iterate through the menu differently in your template).

**Comment by murrant** [comment]
Perhaps, thanks for letting me know that approach is a dead end at least :)

**Comment by atsuzaki** [comment]
Is there still interest in adding this feature?

**Comment by oprypin** [comment]
There's something similar to separators in this theme
https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-sections

and it indeed relies on just an extra nesting level of the indentation. Could you just do the same, or is there something I'm missing about this request?

**Comment by pawamoy** [comment]
Since this seems to be theme-specific, I'll move this to a discussion!
